.gitmodules

```diff
@@ -1,40 +1,40 @@
 [submodule "cdap"]
 	path = cdap
 	url = https://github.com/Guavus/cdap.git
-	branch = develop_guavus
+	branch = release/build_guavus_5.1.2_5.1.217
 [submodule "app-artifacts/hydrator-plugins"]
 	path = app-artifacts/hydrator-plugins
 	url = https://github.com/Guavus/hydrator-plugins.git
-	branch = develop_guavus
+	branch = release/build_guavus_2.1_5.1.217
 [submodule "security-extensions/cdap-security-extn"]
 	path = security-extensions/cdap-security-extn
 	url = https://github.com/Guavus/cdap-security-extn.git
-	branch = develop_guavus
+	branch = release/build_guavus_0.8_5.1.217
 [submodule "app-artifacts/mmds"]
 	path = app-artifacts/mmds
 	url = https://github.com/Guavus/mmds.git
-	branch = develop_guavus
+	branch = release/build_guavus_1.1_5.1.217
 [submodule "app-artifacts/dre"]
 	path = app-artifacts/dre
 	url = https://github.com/Guavus/dre.git
-	branch = develop_guavus
+	branch = release/build_guavus_1.1_5.1.217
 [submodule "cdap-ambari-service"]
 	path = cdap-ambari-service
 	url = https://github.com/Guavus/cdap-ambari-service.git
-  	branch = develop_guavus
+  	branch = release/build_guavus_5.1_5.1.217
 [submodule "sentry"]
 	path = sentry
 	url = https://github.com/Guavus/sentry
-	branch = develop_guavus
+	branch = release/build_guavus_1.7.0_5.1.217
 [submodule "app-artifacts/auto-feature-engineering"]
 	path = app-artifacts/auto-feature-engineering
 	url = https://github.com/Guavus/auto-feature-engineering.git
-        branch = develop
+        branch = release/build_guavus_5.1.217
 [submodule "app-artifacts/cdap-mrds"]
 	path = app-artifacts/cdap-mrds
 	url = https://github.com/Guavus/cdap-mrds.git
-	branch = develop
+	branch = release/build_guavus_5.1.217
 [submodule "app-artifacts/auto-metadata-service"]
 	path = app-artifacts/auto-metadata-service
 	url = https://github.com/Guavus/auto-metadata-service.git
-	branch = develop_guavus
+	branch = release/build_guavus_1.0.0_5.1.217
```

Comment:
The whole file should stay at `develop_guavus` branch. Atleast on cdap-build, there should not be any references of 217 build.